packages/bluesnap-direct-integration/src/bluesnap-direct-constants.ts

```diff
@@ -0,0 +1,55 @@
+import { HostedFieldType } from '@bigcommerce/checkout-sdk/payment-integration-api';
+
+import { BlueSnapDirectHostedFieldTagId as HostedFieldTagId } from './types';
+
+export const HOSTED_FIELD_TYPES: {
+    [HostedFieldTagId.CardNumber]: HostedFieldType.CardNumber;
+    [HostedFieldTagId.CardExpiry]: HostedFieldType.CardExpiry;
+    [HostedFieldTagId.CardCode]: HostedFieldType.CardCode;
+} = Object.freeze({
```

Comment:
They are already `const`, so it's not possible to use `readonly`. Should I remove freeze? I added them to avoid mutations, `const` / `readonly` doesn't prevent the above.

Reply:
As we are using typescript for the whole project, I think compile time checking is enough. Also, we got a unique type name here.

Reply:
Done, I removed `freeze`